sympy/integrals/tests/test_failing_integrals.py

```diff
@@ -1,13 +1,17 @@
 # A collection of failing integrals from the issues.
 
 from sympy import (
-    integrate, Integral, exp, oo, pi, sign, sqrt, sin, cos,
-    tan, S, log, gamma, sinh, sec, zeta, acos, atan, sech, csch
+    integrate, Integral, exp, oo, pi, sign, sqrt, sin, cos, sympify, Piecewise,
+    tan, S, log, gamma, sinh, sec, acos, atan, sech, csch, DiracDelta, sstr
 )
 
 from sympy.utilities.pytest import XFAIL, SKIP, slow, skip, ON_TRAVIS
 
-from sympy.abc import x, k, c, y, R, b, h, a, m, z
+from sympy.abc import x, k, c, y, b, h, a, m, z, n, t
+
+
+def NS(e, n=15, **options):
+    return sstr(sympify(e).evalf(n, **options), full_prec=True)
```

Comment:
This doesn't seem to be used anywhere

Reply:
You are right. It was used in the `test_failing_integrals()` which I moved here, just to realize that the text was not failing at all anymore, just badly written. I'll remove it.